src/app/home/home.component.html

```diff
@@ -0,0 +1,81 @@
+<!-- <div>
+    <ul>
+        <li>
+            Event Name
+        </li>
+        <li>
+            Date 
+        </li>
+        <li>
+            Time 
+        </li>
+        <li>
+            Address  
+        </li>
+        <li>
+            City 
+        </li>
+        <li>
+            Country 
+        </li>
+        <li>
+            Action
+        </li>
+        <button (click)="back()"> Register More</button>
+    </ul>
+    <div *ngIf="userData != '' ">
+        <ul *ngFor ="let x of userData; index as i">
+            <li>
+                {{x.eventName}}
+            </li>
+            <li>
+                {{x.date}}
+            </li>
+            <li>
+                {{x.time}}
+            </li>
+            <li>
+                {{x.address}}
+            </li>
+            <li>
+                {{x.city}}
+            </li>
+            <li>
+                {{x.country}}
+            </li>
+            <button (click)="editData(userData,i)">Edit</button>
+            <button (click)="deleteData(userData,i)">Delete</button>
+        </ul>
+    </div>
+    <p *ngIf="userData == '' ">No data found</p>
+</div> -->
+<table class="table">
+    <thead>
+      <tr>
+        <th scope="col">S.No</th>
+        <th scope="col">EventName</th>
+        <th scope="col">Date</th>
+        <th scope="col">Time</th>
+        <th scope="col">Address</th>
+        <th scope="col">City</th>
+        <th scope="col">Country</th>
+        <th scope="col">Action</th>
+      </tr>
+    </thead>
+    <tbody style="background: gainsboro;">
+      <tr  *ngFor ="let x of userData; index as i">
+        <th scope="row">{{i+1}}</th>
+        <td>{{x.eventName}}</td>
+        <td>{{x.date}}</td>
+        <td>{{x.time}}</td>
+        <td>{{x.address}}</td>
+        <td>{{x.city}}</td>
+        <td>{{x.country}}</td>
+        <td><button class="btn btn-primary ms-1" (click)="editData(userData,i)">Edit</button>
```

Comment:
Why are we sending userData in the function? There are  better approach for doing the same